source/_components/dominos.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Dominos Pizza"
+description: "Instructions how to setup Dominos Pizza ordering within Home Assistant."
+date: 2017-11-05 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: dominos.png
+ha_category: Other
+ha_version: 0.57
+---
+
+The `Dominos` component allows you to order Dominos Pizza from within your Home Assistant scripts and automations.
+
+At present, this component only supports ordering within Canada and the US.
+
+To enable the component, you need to set up your customer information and define some orders.
+
+Orders are a group of product codes. You can get these product codes by inspecting an order request from the Dominos web app, or you can [add this custom panel by following this readme](https://github.com/wardcraigj/hass-dominos-panel) to see the available product codes in a separate panel in your install.
+
+Currently, there is no support in this component for toppings, coupons or order tracking.
+
+```yaml
+dominos:
+  country_code: ca
+  first_name: Justin
+  last_name: Trudeau
+  email: justin.trudeau@parl.gc.ca
+  phone: 6139950253
+  address: 24 Sussex Dr, Ottawa, ON, K1M1M4
+  orders:
+  - name: Medium Pan
+    codes:
+      - P12IPAZA
+```
+
+Now you can use the Dominos service to order pizza within your automations:
+
+```yaml
+- service: dominos.order
+  data:
+    order_entity_id: dominos.medium_pan
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tasks for this part of the documentation.
For more information see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
@wardcraigj You need to remove this header as well